packages/elements/src/icon/index.ts

```diff
@@ -113,16 +114,11 @@ export class Icon extends BasicElement {
   }
 
   /**
-   * Called after the component is first rendered
-   * @param changedProperties Properties which have changed
+   * Called when connected to DOM
    * @returns {void}
    */
-  protected firstUpdated (changedProperties: PropertyValues): void {
-    super.firstUpdated(changedProperties);
-    /**
-     * We have to call this here because
-     * polyfilled browsers only get variables at this point.
-     */
+  public connectedCallback (): void {
+    super.connectedCallback();
     this.setPrefix();
   }
```

Comment:
This doesn't change much and means we set the prefix on every DOM connection. I'm not sure this is any better than before. Maybe we just leave it as it is?

Reply:
Indeed. I did confirm with others, let's keep it as is.